Bixal/move.mil
# Update drupal core

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

- Updates Drupal core and other libraries
- Fixes severity vulnerabilities on move_mil theme
- Fixes severity vulnerabilities on react tools

## Testing

To verify the changes proposed in this pull request…

1. Remove vendor content  and mode_module
1. run composer install
1. run drush updatedb
1. Go to theme directory and run `npm install && npm run build`
1. Go to tools directory and run `npm install && npm run build`
1. clear cache
1. Verify that the site is still running correctly
